Allow trigger side filtering on aggregation keys (#1149)
WICG/attribution-reporting-api: AGGREGATE.md

true_agg_geo_value = 1024 * raw_agg_geo_value / (L1 / 2)
```

Note: The `filters` field will still apply to aggregatable reports, and each
dict in `aggregatable_trigger_data` can still optionally have filters applied
to it just like for event-level reports.

Note: The `aggregatable_values` field may also be specified as a list of dict,
where each dict contains a dict of key-value pair as outlined above as well as

> an optional `filters` (and its corresponding `not_filters`) field, allowing

Reviewer: ```suggestion
an optional `filters` and an optional `not_filters` fields, allowing
```

trigger registrations to customize how values are contributed to keys depending

> on the context of the source.

Reviewer: ```suggestion
on source event data.
```

> on the context of the source.

Reviewer: Please also mention that if multiple values match, then the first matching one will be used.

```jsonc
{

>   ...

Reviewer: ```suggestion
  ...,
```

true_agg_geo_value = 1024 * raw_agg_geo_value / (L1 / 2)
```

Note: The `filters` field will still apply to aggregatable reports, and each
dict in `aggregatable_trigger_data` can still optionally have filters applied
to it just like for event-level reports.

> Note: The `aggregatable_values` field may also be specified as a list of dict,
> where each dict contains a dict of key-value pair as outlined above as well as

Reviewer 2: ```suggestion
Note: The `aggregatable_values` field may also be specified as a list of dictionaries,
where each dictionary contains a dictionary `values` of key-value pairs as outlined above as well as
```

optional `filters` and `not_filters` fields, allowing trigger registrations to

> customize how values are contributed to keys depending on source event data. If

Reviewer 2: ```suggestion
customize how values are contributed to keys depending on source filter data. If
```

multiple list entries have filters that match the source's filters, only the

> first entry and its corresponding values will be used.

Reviewer 2: ```suggestion
first entry and its corresponding values will be used.

For example:
```
